packages/manager/apps/telecom/src/app/telecom/telephony/line/phone/accessories/choice/telephony-line-phone-accessories-choice.html

```diff
@@ -0,0 +1,162 @@
+                </oui-dropdown-content>
+            </oui-dropdown>
+            <oui-dropdown data-ng-if="$ctrl.phone">
+                <oui-dropdown-trigger
+                    text="{{ ::'telephony_line_phone_accessories_filter_compatibility' | translate }}"
+                >
+                </oui-dropdown-trigger>
+                <oui-dropdown-content>
+                    <oui-dropdown-item data-on-click="$ctrl.filterWithPhone()">
+                        <span
+                            data-translate="telephony_line_phone_accessories_filter_compatibility_compatible"
+                        ></span>
+                    </oui-dropdown-item>
+                    <oui-dropdown-item data-on-click="$ctrl.filterAll()">
+                        <span
+                            data-translate="telephony_line_phone_accessories_filter_compatibility_all"
+                        ></span>
+                    </oui-dropdown-item>
+                </oui-dropdown-content>
+            </oui-dropdown>
+
+            <!-- Define tabs -->
+            <oui-tabs>
+                <oui-tabs-item
+                    data-ng-repeat="brand in $ctrl.brandList"
+                    data-heading="{{ ::brand }}"
+                    data-on-active="$ctrl.filterByBrand(brand)"
+                >
+                    <!-- Define the thumbnail for accessories list -->
+                    <div class="row">
+                        <div
+                            class="col-xs-12 col-sm-6 col-md-3 accessory-item"
+                            data-ng-repeat="accessory in $ctrl.accessoriesDisplayed"
+                        >
+                            <oui-radio-group
+                                data-model="$ctrl.accessoryToOrder"
+                                data-on-change="$ctrl.selectAccessory()"
+                            >
+                                <oui-radio
+                                    thumbnail
+                                    variant="light"
+                                    data-value="accessory.name"
+                                >
+                                    <div class="thumbnail">
+                                        <img
+                                            data-ng-if="accessory.img"
+                                            alt=""
+                                            data-ng-alt="{{accessory.description}}"
+                                            data-ng-src="{{accessory.img}}"
+                                        />
+                                        <span
+                                            data-ng-if="!accessory.img"
+                                            class="text-center ovh-font ovh-font-phone"
+                                        ></span>
+                                    </div>
+                                    <div class="text-center">
+                                        {{accessory.description}}
```

Comment:
Please, to avoid display during loading
```suggestion
                                        <span data-ng-bind="accessory.description"></span>
```